src/utils/index.js

```diff
@@ -0,0 +1,53 @@
+
+export const mapComment = commentLen => {
+  if (commentLen > 2) {
+    return `${commentLen} comments`;
+  } else if (commentLen === 1) {
+    return `${commentLen} comment`;
+  } else {
+    return `discuss`;
+  }
```

Comment:
What happens when there are exactly two comments?

Also, small nitpick: If you return in your `if` branches, you don't need an `else` statement (like you do in `mapToTime` below)

Reply:
updated